scripts/gff_metaparser/gen_meta_conf.py

```diff
@@ -63,6 +63,8 @@ def get_args():
                       type=str, default = "EnsemblMetazoa", help="species.division default")
   parser.add_argument("--genebuild_method", metavar="import", required = False,
                       type=str, default = "import", help="genebuild.method default")
+  parser.add_argument("--genebuild_method_display", metavar="Import", required = False,
+                      type=str, default = "Import", help="genebuild.method_display default")
```

Comment:
```suggestion
  parser.add_argument("--genebuild_method_display", metavar="Import", required=False,
                      type=str, default="Import", help="genebuild.method_display default")
```
To make it PEP8 compliant :upside_down_face: